CONTRIBUTING.md

```diff
@@ -133,6 +133,7 @@ We appreciate contributions via pull requests! Before sending a pull request, en
 1. You are working against the latest source on the _master_ branch.
 2. You check existing open, and recently merged, pull requests to make sure someone else hasn't addressed the problem already.
 3. You open an issue to discuss any significant work - we would hate for your time to be wasted.
+4. You have installed the [AWS `git secrets`](https://github.com/awslabs/git-secrets) hook, which will ensure you aren't accidentally commiting AWS access or secret keys. This is enforced on commit.
```

Comment:
I would instead just add it to the list here: https://github.com/aws/aws-toolkit-vscode/blob/2eb45825b784fc1bdb778d644a004671de1999c9/CONTRIBUTING.md#setup

Reply:
Works for me, I was on the fence considering you don't need it to build, just to commit.

Reply:
yeah. But you also don't need git :) Better to keep everything in 1 place.